addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -13,13 +13,12 @@
   },
   "private": true,
   "dependencies": {
-    "angular2": "2.0.0-beta.9",
-    "es6-promise": "^3.0.2",
```

Comment:
There was a few betas around 5-7 if I remember right that needed it. Alex would know exactly which.